crates/libs/bindgen/Cargo.toml

```diff
@@ -21,7 +21,7 @@ default = ["metadata"]
 metadata = []
 
 [dependencies]
-windows-metadata = { path = "../metadata", version = "0.55" }
+windows-metadata = { path = "../metadata", version = "0.56.0" }
```

Comment:
Just heads up, this dependency now extends into .PATCH which may/may not be intentional.